### PR TITLE
Scrape minimum grade prerequisites

### PR DESCRIPTION
**Pull request by powe97**

### CODE REVIEW

File: scrapers/sis_scraper/prerequisites.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def tokenize(prerequisites: str) -> Tokens:
         ("OR", r"or"),
         ("AND", r"and"),
         ("COURSE", r"[a-zA-Z]{4}(?:-| )\d{4}"),
+        ("MIN_GRADE", r"(?<=Minimum Grade of )[A-Z]*[+-]?"),
```
Comment (benjaminrsherman) [inline scrapers/sis_scraper/prerequisites.py:28]: This should use `[A-Z]+` right?

Comment (powe97) [reply scrapers/sis_scraper/prerequisites.py:28]: Yes, oops.

File: scrapers/sis_scraper/prerequisites.py [hunk 2]
```diff
@@ -55,10 +56,14 @@ def parse_atom(tokens: Tokens, cur_tok: int):
         result, cur_tok = parse_tokens(tokens, cur_tok + 1)
         assert tokens[cur_tok][0] == "CLOSE_PAREN"
     else:
+        min_grade_exists = tokens[cur_tok + 1][0] == "MIN_GRADE"
```
Comment (benjaminrsherman) [inline scrapers/sis_scraper/prerequisites.py:59]: What if this is the last token?

Comment (powe97) [reply scrapers/sis_scraper/prerequisites.py:59]: Good point­– this did not happen in the entire data set but I should add a check for that.

Comment (benjaminrsherman) [reply scrapers/sis_scraper/prerequisites.py:59]: Jk, there's the END token

Comment (powe97) [reply scrapers/sis_scraper/prerequisites.py:59]: Nvm looks like it's actually impossible for that to happen due to END existing

**Review by benjaminrsherman — APPROVED** [review]

**Pull request merged**